rest/app.js

```diff
@@ -0,0 +1,108 @@
+const insertUsersInTable = (html) => {
+  document.querySelector("table").innerHTML = html;
+};
+const refresh = () => {
+  window.location.reload();
+};
+const displayEditBox = () => {
+  document.querySelector("#editNotification").innerHTML = `
+    <div id="editContainer">
+        <div id="editBox">
+        <input type="text" placeholder="Name" id="newName">
+        <input type="text" placeholder="last name" id="newLastName">
+        <button id="update">Update</button>
+    </div>
+</div>`;
+};
+
+const hideEditBox = () => {
+  document.querySelector("#editNotification").innerHTML = "";
+};
+
+const fetchUsers = async () => {
```

Comment:
this method is not only fetching users, here you have fetch and data population, separate dom population to other method